src/app/(auth)/signup/components/SignUpDetails.js

```diff
@@ -1,12 +1,22 @@
 "use client";
 
-import { FormikTextField } from "@/app/activities/[activityId]/edit/components/formikFields";
+import { FormikTextField, createHandleSubmit, FormikErrors } from "@/app/activities/[activityId]/edit/components/formikFields";
 import { Button, Container, Typography } from "@mui/material";
 import { Form, Formik } from "formik";
 import * as Yup from "yup";
 import { SignUpContainer } from "./SignUpAccount";
+import { useMutation } from "react-query";
+import { signupDetails } from "@/app/api.mjs";
 
 export default function SignUpDetails({ goToNextStep }) {
+
+  const mutation = useMutation(signupDetails);
```

Comment:
If authenticated user will enter this page accidentally, he will got an error.
What is the best way to handle it?